### ISSUE TITLE
Aggregate JavaScript Breaks Site

### ISSUE THREAD

**Issue by ghost** [forumone/gesso#313]
Hi there, not exactly sure if this a Gesso thing or not, but thought I would post it here to see if anyone has this same issue. If I check "Aggregate JavaScript files" in Drupal, a few things on the site break including the admin toolbar. They work fine with that option unchecked. Checking the console, I see the following error:
```
SyntaxError: "" string literal contains an unescaped line break
```

Which if I look into the compiled JS and find the line number (I think this is the one) I see this area:

```
/***/ "./node_modules/core-js/internals/a-function.js":
/*!******************************************************!*\
  !*** ./node_modules/core-js/internals/a-function.js ***!
  \******************************************************/
/*! no static exports found */
/***/ (function(module, exports) {

eval("module.exports = function (it) {\n  if (typeof it != 'function') {\n    throw TypeError(String(it) + ' is not a function');\n  } return it;\n};\n//# sourceURL=[module]\n//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIndlYnBhY2s6Ly8vLi9ub2RlX21vZHVsZXMvY29yZS1qcy9pbnRlcm5hbHMvYS1mdW5jdGlvbi5qcz8xYzBiIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBO0FBQ0E7QUFDQTtBQUNBLEdBQUc7QUFDSCIsImZpbGUiOiIuL25vZGVfbW9kdWxlcy9jb3JlLWpzL2ludGVybmFscy9hLWZ1bmN0aW9uLmpzLmpzIiwic291cmNlc0NvbnRlbnQiOlsibW9kdWxlLmV4cG9ydHMgPSBmdW5jdGlvbiAoaXQpIHtcbiAgaWYgKHR5cGVvZiBpdCAhPSAnZnVuY3Rpb24nKSB7XG4gICAgdGhyb3cgVHlwZUVycm9yKFN0cmluZyhpdCkgKyAnIGlzIG5vdCBhIGZ1bmN0aW9uJyk7XG4gIH0gcmV0dXJuIGl0O1xufTtcbiJdLCJzb3VyY2VSb290IjoiIn0=\n
/***/ }),
```

The line in question is the eval line specifically. Since it's referencing node-modules, is this part of gesso? Thanks!

**Comment by ghost** [comment]
Just wanted to follow up on this one - the above was tested using an older version of Gesso and I just started a new site with the latest version of Gesso and I'm getting the same thing happening - if aggregate is turned on for JS, it seems to break the front-end (toolbar and other scripts). Wanted to make sure you were aware that it was still a thing with a newer version. Thanks for looking into it!

**Comment by kmonahan** [comment]
Thanks for the additional details, @dutton242! We'll look into this.

One question for you so that I can try to replicate on my end: are you running `gulp` or `gulp build` when you're seeing the error?

**Comment by ghost** [comment]
Ahh, good call! I was using just "gulp" and having it watch files and such. However, I just tried rebuilding with "gulp build" and that seems to have fixed it! So, seems the issue is when just "gulp" is run by itself. Does that help?

**Comment by dcmouyard** [comment]
Using `gulp build` fixed the issue for me too!

**Comment by dcmouyard** [comment]
This was fixed in #319
